iroh-net/src/hp/netcheck.rs

```diff
@@ -120,6 +122,22 @@ impl Report {
     pub fn any_port_mapping_checked(&self) -> bool {
         self.upnp.is_some() || self.pmp.is_some() || self.pcp.is_some()
     }
+
+    /// Updates the portmap related values based on a [`portmapper::ProbeResult`].
+    pub fn update_portmap_probe(&mut self, probe_result: Option<portmapper::ProbeOutput>) {
+        match probe_result {
+            Some(portmapper::ProbeOutput { upnp, pmp, pcp }) => {
```

Comment:
Suggestion, lest just just store the `ProbeOutput` instead of these three individual fields.

Reply:
done in 0846437